RockLib.HealthChecks.AspNetCore.ResponseWriter/RockLibHealthChecks.cs

```diff
@@ -35,20 +49,24 @@ private async static Task WriteResponse(HttpContext httpContext, HealthReport he
             {
                 var entry = x.Value;
 
-                var result = new HealthCheckResult()
+                var result = new HealthCheckResult
                 {
                     ComponentName = GetComponentName(x.Key),
                     MeasurementName = GetMeasurementName(x.Key),
                     Status = MapStatus(entry.Status),
-                    Output = entry.Description,
-                    ["exception"] = entry.Exception?.ToString(),
                     ["duration"] = entry.Duration
                 };
 
+                if (!HideExceptions)
+                    result["exception"] = entry.Exception?.ToString();
+
+                if (!HideOutputs)
+                    result.Output = entry.Description;
+
                 // TODO: Detect collisions between entry.Exception/result.Output,
                 // entry.Description/result["description", or entry.Duration/result["duration"].
 
-                foreach (var data in entry.Data)
+            foreach (var data in entry.Data)
```

Comment:
Looks like this was accidentally outdented.
```suggestion
                foreach (var data in entry.Data)
```